CloudSQLReplicator/google/cloud/pontem/sql/replicator/util/gcp_api_util.py

```diff
@@ -88,10 +93,91 @@ def build_sql_admin_service(credentials=None):
         credentials or default_credentials
     )
 
+    service = discovery.build(
+        service,
+        version,
+        http=authorized_http
+    )
+
+    return service
+
+# Compute
```

Comment:
"# Compute" is not a useful or informing comment

Reply:
Fixed.